src/@next/EmptyState/EmptyState.tsx

```diff
@@ -34,26 +33,58 @@ export const EmptyState = ({
   fullWidth = true,
   helpText,
   imageName,
+  image,
   title,
 }: EmptyStateProps) => {
   const displayButtons = !!primaryButtonAction || !!basicButtonAction;
 
-  const isValidImageName = imageNames.includes(imageName);
-  if (imageName && !isValidImageName) {
-    console.warn(`imageName "${imageName}" is not a valid Image Name.`);
+  if (imageName) {
+    console.warn(
+      'imageName will soon be deprecated. Please use image prop instead'
+    );
+
+    if (!imageNames.includes(imageName)) {
+      console.warn(`image "${imageName}" is not a valid Image Name.`);
+    }
+  }
+
+  if (typeof image === 'string' && !imageNames.includes(imageName)) {
+    console.warn(`image "${imageName}" is not a valid Image Name.`);
   }
 
+  const renderImage = () => {
+    if (imageName || typeof image === 'string') {
```

Comment:
can we do a check check if image is present, currently we only do a typeof check this is gonna be true even when we pass an empty string